cellengine/resources/gate.py

```diff
@@ -109,8 +108,6 @@ def create(
         label: List[float] = [],
         gid: str = None,
         locked: bool = False,
-        parent_population_id: str = None,
-        parent_population: str = None,  # TODO
```

Comment:
and creating gates with `createPopulation=true` still requires `parentPopulationId`, see #125 description.

Reply:
Gate creation presents a bit of trouble here, but I've re-jiggered things such that gates are built (with the `.create` method) with an optional `parent_population_id`, and then when the gate is actually created (in `APIClient.create`) then `createPopulation=True` is passed along if fcs_file_id does not exist in the Gate.

This isn't 100% as flexible as the API could provide, but it's the best solution I can think of that allows for multiple gate creation.